src/grid/TraversabilityGrid.cpp

```diff
@@ -66,11 +69,22 @@ const TraversabilityClass& TraversabilityGrid::getTraversability(size_t x, size_
     return traversabilityClasses[cell.getTraversabilityClassId()];
 }
 
-void TraversabilityGrid::setProbability(float probability, size_t x, size_t y)
+uint8_t TraversabilityGrid::getTraversabilityClassId(std::size_t x, std::size_t y) const
 {
     TraversabilityCell cell = this->GridMap::at(x, y);
```

Comment:
This should be a const reference, in order to avoid that a copy of the cell is done.